File: src/main/telemetry/api/Wrapper.java
package main.telemetry.api;

import java.io.IOException;
import java.net.URL;
import java.util.Scanner;

import main.telemetry.API;

public class Wrapper implements API {


  @Override
  public String run() throws IOException {  
    Scanner sc = new Scanner(returnLink().openStream());

    StringBuffer sb = new StringBuffer();
    while (sc.hasNextLine()) {
      sb.append(sc.nextLine()).append("\n");
    }
    sc.close();
    return sb.toString();
  }

  @Override
  public URL returnLink() throws IOException {
    return new URL("https://exoad.github.io/MusicPlayer/api/version");
  }
  
}

File: kt/app/telemetry/api/Wrapper.kt
package app.telemetry.api

import app.telemetry.API
import kotlin.Throws
import java.io.IOException
import java.io.BufferedWriter
import java.io.FileWriter
import java.lang.StringBuilder
import app.telemetry.FileIntegrity
import app.CLI
import app.global.cli.CliType
import java.net.URL
import java.util.*

class Wrapper : API {
    @Throws(IOException::class)
    override fun run(): String {
        val sc = Scanner(returnLink().openStream())
        val sb = StringBuilder()
        while (sc.hasNextLine()) {
            sb.append(sc.nextLine()).append("\n")
        }
        sc.close()
        return sb.toString()
    }

    @Throws(IOException::class)
    override fun returnLink(): URL {
        return URL("https://exoad.github.io/MusicPlayer4J/api/version")
    }
}